jsHelper/spicetifyWrapper.js

```diff
@@ -566,6 +566,11 @@ window.Spicetify = {
 		}
 	});
 
+	const contextMenuChunk = Object.entries(require.m).find(([, value]) => value.toString().includes("toggleContextMenu"));
```

Comment:
```suggestion
	const contextMenuChunk = chunks.find(([, value]) => value.toString().includes("toggleContextMenu"));
```

wait for #2792 to be merged and change it